headers.asm

```diff
@@ -147,9 +146,24 @@ nt_c_store:
 
 nt_plus_store:
         .byte 2, UF
-        .word nt_execute, xt_plus_store, z_plus_store
+        .word nt_zero, xt_plus_store, z_plus_store
         .text "+!"
 
+nt_zero:
```

Comment:
i noticed that disasm shows literal 0 as "false", so I moved these special literals earlier